src/components/CenteredMap/LoadableCenteredMap.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import Loadable from 'react-loadable'
+
+import Loader from 'common/components/Loader'
+
+const LoadableCenteredMap = Loadable({
+  loader: () => import(/* webpackChunkName: 'components/map' */ './CenteredMap'),
```

Comment:
The comment allows specifying a chunk name for webpack, otherwise it would be just something like `8,js`.